packages/main/src/ColorPalette.ts

```diff
@@ -245,14 +300,24 @@ class ColorPalette extends UI5Element {
 		});
 	}
 
+	_getEffectiveColorItems() {
+		let colorItems = this.colors;
+
+		if (this.popupMode) {
+			colorItems = this.getSlottedNodes<ColorPaletteItem>("colors");
+		}
```

Comment:
Do we need that check? As I know we use getSlottedNodes when something is rendered inside the static area but here we don't.

Reply:
This is needed for consistent behavior handling among ColorPalette and ColorPalettePopover, thats why we get the slottedNodes only when this.popupMode is true;